misc/docerina/src/main/java/org/ballerinalang/docgen/generator/model/Module.java

```diff
@@ -60,4 +62,7 @@
 
     @Expose
     public List<Path> resources = new ArrayList<>();
+
+    public Queue<String> linkedAnonObjects = new LinkedList<>();
```

Comment:
What is the purpose of using a queue ?

Reply:
In the case of an anonymous object function having a anon return(ex- lang.stream->iterator function->return(T8)->next function->returns T7) if we use a an arraylist, a concurrentmodication exception is thrown. This is because when we are iterating through the module.linkedAnonObjects and creating the anon types, another anon type is getting added to the module.linkedAnonObjects.